engine/options_menu.asm

```diff
@@ -373,57 +380,56 @@ Options_Print: ; e4424
 ; e445a
 
 .Strings:
+; entries correspond to PRINT_* constants
```

Comment:
`GBPRINTER_*`

Reply:
No, the `PRINT_*` constants are the 0-5 indexes; `GBPRINTER_*` are the raw brightness values. Likewise for `TEXT_SPEED_*` (indexes) versus `TEXT_DELAY_*` (frame delays).

On second thought, those are bad names. I'm putting `OPT_` in from of the option index constants.